src/Forms/Shared/Samples/Map/BrowseBuildingFloors/readme.md

```diff
@@ -0,0 +1,43 @@
+# Browse building floors
+
+Display and browse through building floors from a floor-aware web map.
+
+![BrowseBuildingFloorsApp](browse-building-floors.png)
```

Comment:
Picture filepath is incorrect. Verify that the screenshot is shown correctly in the description/readme display in the viewer.